hack/test-cmd.sh

```diff
@@ -95,6 +95,8 @@ fi
 GO_OUT="${OS_ROOT}/_output/local/go/bin"
 export PATH="${GO_OUT}:${PATH}"
 
+ln -svf `readlink -f $(which openshift)` `dirname $(which openshift)`/atomic-enterprise
```

Comment:
This should be done by the build (we have an auto linking function in hack/common.sh for OPENSHIFT_BINARY_SYMLINKS - add atomic-enterprise there).  For any of my other comments just add the follow up.